src/Compute/Compute/Generated/Gallery/GalleryGetMethod.cs

```diff
@@ -157,6 +166,55 @@ public void SharedGalleryGet()
             }
         }
 
+        public void CommunityGalleryGet()
+        {
+            if (this.IsParameterBound(c => c.GalleryPublicName))
+            {
+                CommunityGallery result = CommunityGalleriesClient.Get(this.Location, this.GalleryPublicName);
+                var psObject = new PSCommunityGallery();
+                ComputeAutomationAutoMapperProfile.Mapper.Map<CommunityGallery, PSCommunityGallery>(result, psObject);
+                WriteObject(psObject);
+            }
+            else if(this.IsParameterBound(c=> c.Community))
+            {
+                //find out if its ok to create client locally
+                ResourceGraphClient rgClient = AzureSession.Instance.ClientFactory.CreateArmClient<ResourceGraphClient>(DefaultContext, AzureEnvironment.Endpoint.ResourceManager);
```

Comment:
@isra-fel is this ok to initiate rgClient like this?

Reply:
Yes